packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -124,34 +120,29 @@ export default function LinkPreview( {
 						) }
 					</span>
 				</span>
-
 				<Button
 					icon={ edit }
-					label={ __( 'Edit' ) }
-					className="block-editor-link-control__search-item-action"
+					label={ __( 'Edit link' ) }
 					onClick={ onEditClick }
 					size="compact"
 				/>
 				{ hasUnlinkControl && (
 					<Button
 						icon={ linkOff }
-						label={ __( 'Unlink' ) }
-						className="block-editor-link-control__search-item-action block-editor-link-control__unlink"
+						label={ __( 'Remove link' ) }
 						onClick={ onRemove }
 						size="compact"
 					/>
 				) }
 				<Button
-					icon={ copy }
-					label={ __( 'Copy URL' ) }
-					className="block-editor-link-control__search-item-action block-editor-link-control__copy"
+					icon={ copySmall }
+					label={ __( 'Copy link' ) }
 					ref={ ref }
 					disabled={ isEmptyURL }
 					size="compact"
 				/>
 				<ViewerSlot fillProps={ value } />
 			</div>
-			{ additionalControls && additionalControls() }
```

Comment:
I believe this is in order to remove the `Opens in new tab`.

I have two concerns. The first is technical and the second is based on principle.

Firstly, if we decide we do want to remove the `Opens in new tab` then we should restore this line and instead look in the main `index.js` file for where we _pass_ `additionalControl` to `<LinkPreview>`. The extension point should remain as is.

Secondly, if we are going to remove `Opens in new tab` from the preview _without_ first merging https://github.com/WordPress/gutenberg/pull/58279 then we should do this in it's own PR. 

Why? Because it's a contentious decision and _if_ we need to revert it, it would be easier for release leads to handle in a single PR which does one thing. 

I appreciate it might seem pedantic, but it will make a real different when we're in the full flow of pre-release flight checks 🙏 

Personally now the link preview remains open, I see an argument for removing the checkbox and instead requiring one additional click to change the setting. This is much better than it was previously, when the Link UI would close upon initial creation therefore requiring several clicks to reach the "edit" view again. This new UX is much smoother and so it's less of a concern to have the checkbox available on the preview.

Reply:
> Secondly, if we are going to remove Opens in new tab from the preview without first merging https://github.com/WordPress/gutenberg/pull/58279 then we should do this in it's own PR.

I'm still not certain about #59279 to be honest — https://github.com/WordPress/gutenberg/pull/58279#issuecomment-1915841739

Reply:
> feels lighter, because it is.

Based on my experience I would tend to agree.

I would still recommend we do this bit as a _separate_ PR 🙏

Reply:
I removed this from this PR. It can be done in a follow-up.